### PR TITLE
Upgrade Angular

### PR DESCRIPTION
**Pull request by Hinton**
I forgot to upgrade angular2-toaster and angularstic in web. They are updated here.

### CODE REVIEW

**Comment by Hinton** [comment]
Need to do some final testing before merging.

**Comment by Hinton** [comment]
Both build:watch and build:prod seems to work. A quick testing shows no broken features.

File: src/popup/app.component.ts [hunk 1]
```diff
@@ -67,7 +67,9 @@ export class AppComponent implements OnInit {
         private i18nService: I18nService, private router: Router,
         private stateService: StateService, private messagingService: MessagingService,
         private changeDetectorRef: ChangeDetectorRef, private ngZone: NgZone,
-        private sanitizer: DomSanitizer, private platformUtilsService: PlatformUtilsService) { }
+        private sanitizer: DomSanitizer, private platformUtilsService: PlatformUtilsService) {
+            angulartics2GoogleAnalytics.startTracking();
```
Comment (Hinton) [inline src/popup/app.component.ts:71]: Newer versions of angulartics require explicit calling to `startTracking`.

Comment (kspearrin) [reply src/popup/app.component.ts:71]: Doesn't really matter since this stuff is disabled in other parts of the code.

**Review by kspearrin — DISMISSED** [review]

**Pull request merged**